core/Azure.Mcp.Core/src/Extensions/OpenTelemetryExtensions.cs

```diff
@@ -139,7 +139,7 @@ private static void EnableAzureMonitor(this IServiceCollection services)
     /// </summary>
     /// <param name="entryAssembly">The entry assembly to extract name and version information from.</param>
     /// <returns>A version string.</returns>
-    internal static string GetServerVersion(Assembly entryAssembly)
+    public static string GetServerVersion(Assembly entryAssembly)
```

Comment:
Can you move this to a utility class? of some sort because it has a more general purpose now that we also use it not for telemetry.